### PR TITLE
refactor: remove unused code and correct the docs for logger plugins

### PR DESCRIPTION
**Pull request by Vacant2333**
### Description

Fixes https://github.com/apache/apisix/issues/10494

Remove the unused code and correct the docs for logger plugins, the default meta-log-format will be reomoved.

the loggers's output will be full log with no changes

### Checklist

- [x] I have explained the need for this PR and the problem it solves
- [x] I have updated the documentation to reflect this change
- [x] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

File: docs/en/latest/plugins/clickhouse-logger.md [hunk 1]
```diff
@@ -33,22 +33,22 @@ The `clickhouse-logger` Plugin is used to push logs to [ClickHouse](https://clic
...
+| include_req_body_expr  | array      | False    |                     |               | Filter for when the `include_req_body` attribute is set to `true`. Request body is only logged when the expression set here evaluates to `true`. See [lua-resty-expr](https://github.com/api7/lua-resty-expr) for more.        |
+| include_resp_body      | boolean    | False    | false               | [false, true] | When set to `true` includes the response body in the log.                                                                                                                                                                      |
+| include_resp_body_expr | array      | False    |                     |               | Filter for when the `include_resp_body` attribute is set to `true`. Response body is only logged when the expression set here evaluates to `true`. See [lua-resty-expr](https://github.com/api7/lua-resty-expr) for more.      |
```
Comment (monkeyDluffy6017) [inline docs/en/latest/plugins/clickhouse-logger.md:51]: Why do you reformat these, i can't tell what you changed

**Comment by monkeyDluffy6017** [comment]
Maybe you should keep the doc's format

**Comment by Vacant2333** [comment]
> Maybe you should keep the docs' format

i have remove the format of the tables

**Review by luoluoyuyu — APPROVED** [review]

**Comment by monkeyDluffy6017** [comment]
Please make the ci pass

**Review by monkeyDluffy6017 — APPROVED** [review]

**Comment by Vacant2333** [comment]
@shreemaan-abhishek can u help me to review

**Review by moonming — APPROVED** [review]

**Pull request merged**